packages/opentelemetry-host-metrics/test/metric.test.ts

```diff
@@ -15,7 +15,7 @@
  */
 
 const SI = require('systeminformation');
-import { MetricAttributes } from '@opentelemetry/api-metrics';
+import { MetricAttributes } from '@opentelemetry/api';
```

Comment:
Should it use [`Attributes` instead of `MetricAttributes`](https://github.com/open-telemetry/opentelemetry-js/blob/d1621d2c8bb21484e72706834b53ad0c7da73ca6/api/src/metrics/Metric.ts#L86)?

Since this package now peer depends on `@opentelemetry/api@^1.3.0`, there is no reason to use the old type.

Reply:
done